external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaSourceProviderSuite.scala

```diff
@@ -22,60 +22,99 @@ import java.util.Locale
 import scala.collection.JavaConverters._
 
 import org.mockito.Mockito.{mock, when}
-import org.scalatest.{BeforeAndAfterEach, PrivateMethodTester}
+import org.scalatest.{PrivateMethodTester}
 
 import org.apache.spark.{SparkConf, SparkEnv, SparkFunSuite}
 import org.apache.spark.sql.sources.v2.reader.Scan
 import org.apache.spark.sql.util.CaseInsensitiveStringMap
 
 class KafkaSourceProviderSuite extends SparkFunSuite with PrivateMethodTester {
 
+  private val expected = "1111"
   private val pollTimeoutMsMethod = PrivateMethod[Long]('pollTimeoutMs)
   private val maxOffsetsPerTriggerMethod = PrivateMethod[Option[Long]]('maxOffsetsPerTrigger)
+  private val maxOffsetFetchAttemptsMethod = PrivateMethod[Int]('maxOffsetFetchAttempts)
```

Comment:
I know it's a bit unrelated but can we use similar way like https://github.com/apache/spark/pull/24967#discussion_r304784506 and remove those reflections? I would feel pretty said it some tests failed when I change the name of private variables ..

Reply:
If we would like to allow having it non-private for testing, I'll take care of this after merging. @gaborgsomogyi is on vacation (as he mentioned in other PR) so it would take couple of weeks when we would like to add the change on the scope of this PR.

Reply:
Can you pick up the commit and open a PR to be authored just in case some more comment come from @zsxwing and @cloud-fan.

Reply:
Actually I'm on vacation as well (I just found small amount of time to get back), so let's wait for @gaborgsomogyi to address further comments.

Reply:
Fixed.